projects/canopy/src/lib/card/docs/promotions/guide.stories.mdx

```diff
@@ -0,0 +1,513 @@
+        <lg-card-content>
+          <h3 lgMarginBottom="xxs" class="lg-font--expressive">{{ title }}</h3>
+          <p lgMarginBottom="lg">{{ text }}</p>
+          <a href="#">{{ buttonText }}</a>
+        </lg-card-content>
+      </lg-card>
+    </div>
+    <div lgColSm="12" lgColLg="4">
+      <lg-card
+        lgShadow
+        lgMarginBottom="lg"
+        variant="promotion"
+        [orientation]="{ sm: 'vertical', md: 'horizontal', lg: 'vertical' }">
+        <lg-card-hero-img>
+          <lg-brand-icon [name]="iconName" size="xs"></lg-brand-icon>
+        </lg-card-hero-img>
+        <lg-card-content>
+          <h3 lgMarginBottom="xxs" class="lg-font--expressive">{{ title }}</h3>
+          <p lgMarginBottom="lg">{{ text }}</p>
+          <a href="#">{{ buttonText }}</a>
+        </lg-card-content>
+      </lg-card>
+    </div>
+    <div lgColSm="12" lgColLg="4">
+      <lg-card
+        lgShadow
+        lgMarginBottom="lg"
+        variant="promotion"
+        [orientation]="{ sm: 'vertical', md: 'horizontal', lg: 'vertical' }">
+        <lg-card-hero-img>
+          <lg-brand-icon [name]="iconName" size="xs"></lg-brand-icon>
+        </lg-card-hero-img>
+        <lg-card-content>
+          <h3 lgMarginBottom="xxs" class="lg-font--expressive">{{ title }}</h3>
+          <p lgMarginBottom="lg">{{ text }}</p>
+          <a href="#">{{ buttonText }}</a>
+        </lg-card-content>
+      </lg-card>
+    </div>
+  </div>
+  <div lgRow>
+    <div lgCol="12">
+      <lg-separator variant="dotted" lgMarginTop="none"></lg-separator>
+    </div>
+    <div lgColSm="12" lgColLg="6">
+      <lg-card
+        lgShadow
+        lgMarginBottom="lg"
+        variant="promotion"
+        [orientation]="{ sm: 'vertical', md: 'horizontal', lg: 'vertical' }">
+        <lg-card-hero-img>
+          <lg-brand-icon [name]="iconName" size="xs"></lg-brand-icon>
+        </lg-card-hero-img>
+        <lg-card-content>
+          <h3 lgMarginBottom="xxs" class="lg-font--expressive">{{ title }}</h3>
+          <p lgMarginBottom="lg">{{ text }}</p>
+          <a href="#">{{ buttonText }}</a>
+        </lg-card-content>
+      </lg-card>
+    </div>
+    <div lgColSm="12" lgColLg="6">
+      <lg-card
+        lgShadow
+        lgMarginBottom="lg"
+        variant="promotion"
+        [orientation]="{ sm: 'vertical', md: 'horizontal', lg: 'vertical' }">
+        <lg-card-hero-img>
+          <lg-brand-icon [name]="iconName" size="xs"></lg-brand-icon>
+        </lg-card-hero-img>
+        <lg-card-content>
+          <h3 lgMarginBottom="xxs" class="lg-font--expressive">{{ title }}</h3>
+          <p lgMarginBottom="lg">{{ text }}</p>
+          <a href="#">{{ buttonText }}</a>
+        </lg-card-content>
+      </lg-card>
+    </div>
+  </div>
+</div>
+```
+## Additional development details
+
+### LgCardComponent
+
+The main body of the promotion card. It's composed by the following components:
+
+- `LgCardHeroImageComponent`
+- `LgCardContent`
+
+#### Inputs
```

Comment:
Just noticed that all of this is what's already in the `card` documentation. I wonder if it would be better to simply add a link to that section instead?

Reply:
Ok, let's leave it as is for now 👍

Reply:
Ohh you know what you could do? Something like the "feedback" section. Basically you can add a file in `lib/card/docs` called `card-hero-image-component.md` and into it add the documentation section for `LgCardHeroImageComponent`. 
Then in both in `lib/card/docs/card/guide.stories.mdx` and `lib/card/docs/promotions/guide.stories.mdx` refer to that by doing:

```
import CardHeroImageComponent from '../card-hero-image-component.md';

...

<CardHeroImageComponent />

...
```

Reply:
Yep, this worked and it looks perfect. Done :)